webapp/channels/src/components/integrations/bots/bots.tsx

```diff
@@ -260,11 +262,16 @@ export default class Bots extends React.PureComponent<Props, State> {
                                 ),
                             }}
                         />
-                        <FormattedMarkdownMessage
-                            id='bots.manage.help2'
-                            defaultMessage={'Enable bot account creation in the [System Console]({siteURL}/admin_console/integrations/bot_accounts).'}
+                        <FormattedMessage
+                            id='bots.help2'
+                            defaultMessage={'Enable bot account creation in the <a>System Console</a>.'}
                             values={{
-                                siteURL: getSiteURL(),
+                                a: (chunks: string) => (
+                                    <a
```

Comment:
For internal links refer this https://github.com/mattermost/mattermost/pull/28438/files?file-filters%5B%5D=.json&file-filters%5B%5D=.tsx&show-viewed-files=true#diff-5d7bfc66b91e68df2079b4180a77bbebc113f61df8d73ec67adccab4c54ee843R57